src/main/java/seedu/address/logic/parser/FindTagCaseInsensitiveCommandParser.java

```diff
@@ -6,31 +6,31 @@
 import java.util.List;
 import java.util.stream.Collectors;
 
-import seedu.address.logic.commands.FindTagsCommand;
+import seedu.address.logic.commands.FindTagCaseInsensitiveCommand;
 import seedu.address.logic.parser.exceptions.ParseException;
-import seedu.address.model.person.PersonTagsContainsTagsPredicate;
+import seedu.address.model.person.PersonTagsContainsCaseInsensitiveTagsPredicate;
 import seedu.address.model.tag.Tag;
 
 /**
  * Parses input arguments and creates a new FindTagsCommand object
```

Comment:
Can update the JavaDocs to be more specific on the object created i.e. `FindTagCaseInsensitiveCommand` object, same for `FindTagCaseSensitiveCommand`